Makefile

```diff
@@ -5,3 +5,25 @@ build-docs:
 	make -C doc/ clean
 	make -C doc/ html-noplot
 	cd doc/ && make view
```

Comment:
I'm of the opinion that these scripted tasks in makefiles should be modeled as Poe scripts in pyproject.toml. Makefiles require another system of build-tools to be present, and it splits up our engineering system logic

Reply:
Sure that makes sense. Do you know how to add the clean commands to poetry Poe? Would it just be the shell commands?

Reply:
We still need the make file for the docs folder tho I think.

Reply:
I've removed the `Makefile`, but certain shell-based operators, such as the "|" command does not work inside poe.